test/fixtures/api/responses/get-balance-sheet.json

```diff
@@ -0,0 +1,68 @@
+{
+  "balances": [
+    {
+      "counterparty": "rKm4uWpg9tfwbVSeATv4KxDe6mpE9yPkgJ",
+      "balances": [
```

Comment:
We should avoid naming that suggests that something is a non-whole component of itself, which is a logical contradiction (balances = balances + counterparty). A one-dimensional array of amounts sounds good to me.